<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>oknoWyboruGraczy</class>
 <widget class="QDialog" name="oknoWyboruGraczy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGraphicsView" name="viewGracz1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>251</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nazwaGracz1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Wpisz nazwę 1 gracza</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTytul">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>901</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Wybierz graczy i planszę</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelGracz1">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>60</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Gracz 1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="nazwaGracz2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Wpisz nazwę 2 gracza</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="viewGracz2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>251</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelGracz2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Gracz 2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="nazwaGracz3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Wpisz nazwę 3 gracza</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="viewGracz3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>251</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelGracz3">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>370</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Gracz 3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="nazwaGracz4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>400</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Wpisz nazwę 4 gracza</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="viewGracz4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>460</y>
     <width>251</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelGracz4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>370</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Gracz 4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGraphicsView" name="viewPlansza">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>90</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNowaPlansza">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>520</y>
     <width>340</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Stwórz nową planszę</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGraj">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>570</y>
     <width>340</width>
     <height>90</height>
    </rect>
   </property>
   <property name="text">
    <string>Graj</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelZiarno">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>500</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ziarno generatora</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLewo1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPrawo1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLewo2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPrawo2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLewo3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>660</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPrawo3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>660</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLewo4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>660</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPrawo4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>660</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOstrzezenie">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>670</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="ziarnoGeneratora">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>500</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
